runtimes/eden/src/pallets_util.rs

```diff
@@ -29,11 +29,16 @@ use frame_system::{EnsureRoot, EnsureSigned};
 use primitives::{AccountId, Balance};
 use sp_runtime::Perbill;
 
+parameter_types! {
+	pub const MaxSchedule: u32 = 200;
```

Comment:
Before finalising this number, can we know what's the highest number of the schedules currently in our Eden storage?

Reply:
Yes right now this is stub value, will get the actual as suggested

Reply:
From the existing state of `paradis`, Max length of Vesting schedule is 10, right now configured to double the size 20.

Reply:
@ETeissonniere as Rajesh suggests we can be safe with 20 for the maximum number of grants for individual accounts based on what we have currently stored. However I would like to know your intuition on what would be a good cap here?